win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
How do you want the tests handled?  The tests that I've seen only really test if the options actually get set, rather than test that the option setting did anything.    There are functional tests in node-rcedit now, are these adequate or do you need something else ?